libraries/EEPROM/examples/eeprom_write/eeprom_write.ino

```diff
@@ -33,7 +34,11 @@ void loop() {
   addr = addr + 1;
   if (addr == 512) {
     addr = 0;
-    EEPROM.commit();
+    if (EEPROM.commit()) {
+      Serial.println("EEPROM successfully written");
+    } else {
+      Serial.println("ERROR!  EEPROM write failed");
```

Comment:
committed*
commit*

write is a different operation (from a user's pov).

Reply:
Done